### ISSUE TITLE
Add GELF TCP Support

### ISSUE THREAD

**Issue by sfhorton01** [log4js-node/gelf#1]
UDP is nice, but we have a need for GELF TCP.

**Comment by cula** [comment]
Good idea. I'll add this if i'm available in the future. Any extra work to add this is appreciate.
